src/resources/js/lib/geocoder.ts

```diff
@@ -1,6 +1,26 @@
 import type { MeetingPartialUpdate } from 'bmlt-server-client';
 import { spinner } from '../stores/spinner';
 import { Loader } from '@googlemaps/js-api-loader';
+import { errorModal } from '../stores/errorModal';
+
+let googleMapsHasErrors = false;
+
+if (typeof window !== 'undefined') {
+  const originalConsoleError = console.error;
+  console.error = function (...args) {
```

Comment:
There's got to be a better way to do this than monkey-patching console.error.

Reply:
ya this does smell